### PR TITLE
zigbee-switch: Update white color temp ranges

### PR DESCRIPTION
**Pull request by ctowns**
Updates white color temp ranges with embedded profile config based on vid from former DTH fingerprint.
CHAD-9469: https://smartthings.atlassian.net/browse/CHAD-9469

### CODE REVIEW

**Review by greens — APPROVED** [review]

**Review by greens — APPROVED** [review]
looks good but check the e-mail I just forwarded you from confluence

**Comment by ctowns** [comment]
Reformatted embedded device profiles based on updated format here: https://smartthings.atlassian.net/wiki/spaces/~bflorian/pages/2462811346/Embedded+Device+Configurations

**Review by greens — APPROVED** [review]

**Comment by iplantemn** [comment]
@greens validated that this PR has the correct capability configuration formatting

File: drivers/SmartThings/zigbee-switch/profiles/color-temp-bulb-2200K-4000K.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+  - id: colorTemperature
+    version: 1
+    configuration:
```
Comment (tpmanley) [inline drivers/SmartThings/zigbee-switch/profiles/color-temp-bulb-2200K-4000K.yml:11]: @ctowns `configuration` needs to be changed to `config` in all the profiles per @iplantemn

Comment (ctowns) [reply drivers/SmartThings/zigbee-switch/profiles/color-temp-bulb-2200K-4000K.yml]: Done

**Pull request merged**